frontend/src/consumer/authentication.js

```diff
@@ -0,0 +1,234 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Buffer } from "buffer";
+import CryptoJS from "crypto-js";
+import myImage from "../images/3.jpg";
+
+const encryptionKey =
```

Comment:
Encryption keys like this, should not be exposed in the open source, replace it with place holders like <INSERT_ENCRYPTION_KEY_HERE>.